config/initializers/google.rb

```diff
@@ -0,0 +1 @@
+GOOGLE = YAML.load_file('config/google.yml')[Rails.env].with_indifferent_access['google_key'] if Rails.env.development?
```

Comment:
When you are creating global constants, it's preferrable to be very specific about what they store to avoid namespace classes (like a Google gem).  I'd go back to `GOOGLE_KEY`.  Same with Yelp below.